style.css

```diff
@@ -0,0 +1,35 @@
+* {
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding-top: 100px;
+  justify-content: center;
+}
+
+.main {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  justify-items: center;
+  gap: 1px;
+  width: 300px;
+}
+
+#books {
+  width: 100%;
+}
+
+hr {
+  width: 100%;
+  margin-top: 5px;
+}
+
+p,
+button {
+  font-weight: bold;
+}
```

Comment:
- [ ] For a good user experience kindly make sure to add styles according to the provided `wireframe` as specified in the project requirement. Please use plain `CSS` for styling and make sure your styles match the `wireframe`. See below for more details:
<img width="976" alt="Screenshot 2023-04-25 at 09 32 38" src="https://user-images.githubusercontent.com/107192076/234235963-36976801-78db-4977-8bd2-e9d410da4093.png">